shelley/chain-and-ledger/executable-spec/src/STS/Bhead.hs

```diff
@@ -25,6 +26,9 @@ import           Control.State.Transition
 
 data BHEAD hashAlgo dsignAlgo kesAlgo
 
+data BheadEnv hashAlgo dsignAlgo kesAlgo
+  = BheadEnv Seed (Set.Set (GenKeyHash hashAlgo dsignAlgo))
```

Comment:
maybe we could do an `import Data.Set (Set)` in the module in order to avoid `Set.Set` as type

Reply:
I often do:
```
import           Data,Map.Strict (Map)
import qualified Data,Map.Strict as Map
```

Reply:
I'm a fan of doing :point_up_2: that too. I didn't change any imports in this PR, I just moved things from tuples into data types. I think `Map.Map` and `Set.Set` show up in sever modules. How about I merge this PR now and catch all of the them in a separate PR?